### PR TITLE
AF-2411: Removing pages from Monitoring app

### PR DESCRIPTION
**Pull request by adrielparedes**
Related to:
* https://github.com/kiegroup/appformer/pull/895
* https://github.com/kiegroup/kie-wb-common/pull/3314
* https://github.com/kiegroup/drools-wb/pull/1379
* https://github.com/kiegroup/jbpm-wb/pull/1436
* https://github.com/kiegroup/optaplanner-wb/pull/371
* https://github.com/kiegroup/kie-wb-distributions/pull/1010

### CODE REVIEW

File: business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider [hunk 1]
```diff
@@ -15,6 +15,6 @@
 #
 
 org.uberfire.java.nio.fs.jgit.JGitFileSystemProvider  # file system provider, also default (1st)
-# org.uberfire.java.nio.fs.k8s.K8SFileSystemProvider    # can be set as default at runtime by system property: org.kie.workbench.controller.openshift.enabled
+org.uberfire.java.nio.fs.k8s.K8SFileSystemProvider    # can be set as default at runtime by system property: org.kie.workbench.controller.openshift.enabled
```
Comment (adrielparedes) [inline business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider]: @rhtevan can you confirm this is ok?

**Comment by porcelli** [comment]
what about imposing such a limit just for cloud/k8s?

**Comment by adrielparedes** [comment]
@porcelli sure, let me talk with Evan to see if I have some sys prop to do that

**Comment by rhtevan** [comment]
@adrielparedes Does the changes from this PR still valid and need to be merged?

File: business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider (hunk 1 above)
Comment (rhtevan) [reply business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider:18]: @adrielparedes It is fine to 'enable' k8s implementation now.

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
The error is not related to the changes I commited.

**Comment by adrielparedes** [comment]
jenkins retest this

File: business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
+org.uberfire.java.nio.fs.k8s.K8SFileSystemProvider    # can be set as default at runtime by system property: org.kie.workbench.controller.openshift.enabled
```
Comment (rhtevan) [inline business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider:17]: @adrielparedes When enabling k8sfs, we will still need to keep jGit provider listed as the first as per this [implementation](https://github.com/kiegroup/appformer/blob/0d8c104f2bdb182ca2768eaa5d3a18e57f6f8d1a/uberfire-nio2-backport/uberfire-nio2-api/src/main/java/org/uberfire/java/nio/file/api/FileSystemProviders.java#L79-L87). Second, appreciated if you could update the comments a bit as the name  of the system property as being changes to org.kie.**server**.controller.openshift.enabled, Thanks!

**Comment by mareknovotny** [comment]
jenkins retest this

File: business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider (hunk 1 above)
Comment (rhtevan) [reply business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider]: @adrielparedes  the changes look good to me.

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
The failing test is not related

**Comment by mareknovotny** [comment]
jenkins retest this please


there `Caused by: java.lang.IllegalStateException: zip file closed` due `ERROR [org.uberfire.ext.metadata.io.IndexerDispatcher] (Thread-115) Indexing error for indexer [id=org.uberfire.ext.metadata.io.CoreIndexer]: org.apache.lucene.store.AlreadyClosedException: this IndexWriter is closed`

I saw similar issue with deployment of business-monitoring-webapp.war on different PRs, could we improve that test **org.kie.bc.DeploymentIT.org.kie.bc.DeploymentIT**?

**Comment by mareknovotny** [comment]
reported as https://issues.redhat.com/browse/AF-2535

**Comment by ederign** [comment]
@mareknovotny merging this because we get a green fdb on appformer one. Indeed we need to improve this test.

**Pull request merged**